tests/tests/tier0/monitor-system-status-ctrl-stop/test_monitor_system_status_ctrl_stop.py

```diff
@@ -29,14 +31,22 @@ def exec(ctrl: BluechiControllerContainer, nodes: Dict[str, BluechiNodeContainer
     if result != 0:
         raise Exception(f"Failed to stop bluechi-controller: {output}")
 
-    # wait a bit to process all events
-    time.sleep(1)
-
-    result, output = ctrl.exec_run("cat /tmp/events")
-    if result != 0:
-        raise Exception(f"Failed to get events file: {output}")
-
-    assert output == "down,"
+    LOGGER.info("Checking events processed by the system monitor...")
+    expected_events = ["down"]
+    while True:
```

Comment:
Shouldn't we add some timeout into the loop?

Reply:
The timeout from [conftest.py](https://github.com/eclipse-bluechi/bluechi/blob/main/tests/conftest.py#L17) will be used - so after 45s the test is cancelled anyway.